Automattic/jetpack
# AI Proofread: Toggle feature and list features from configuration

Ref: https://github.com/orgs/Automattic/projects/667/views/24?pane=issue&itemId=70344145
Ref: https://github.com/orgs/Automattic/projects/667/views/24?pane=issue&itemId=70344201

## Proposed changes:

* Remove unused css from `breve.scss`.
* List sidebar features from `features.configuration`.
* Toggle specific feature.
* Update logic for feature color.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Open your editor and make sure you have Breve enabled.
* Write some post that has complex words.
* It should be highlighted.
* Disable the Complex words feature.
* They should disappear.
* Reload the page.
* They should stay disabled.
